spec/models/comment_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+describe Comment do
+  let!(:comment) { FactoryGirl.create(:comment) }
+  it "sends an email" do
+    expect { comment.notification_mailer 
+      }.to change { ActionMailer::Base.deliveries.count }.by(1) 
```

Comment:
`end` at 7, 6 is not aligned with `expect { comment.notification_mailer ` at 6, 4<br>Trailing whitespace detected.